docs/instrumentation.md

```diff
@@ -137,11 +141,53 @@ AnyCable also supports emitting real-time metrics to [StatsD](https://github.com
 For that, you must specify the StatsD server UDP host:
 
 ```sh
-anycable-go -statsd_host=localhost:8125
+anycable-go --statsd_host=localhost:8125
 ```
 
 Metrics are pushed with the `anycable_go.` prefix by default. You can override it by specifying the `statsd_prefix` parameter.
 
+```sh
+# HELP anycable_go_clients_num The number of active clients
+# TYPE anycable_go_clients_num gauge
```

Comment:
HELP and TYPE are from Prometheus output, let's remove them.

StatsD payload looks differently. You can see it by running a [fake StatsD server](https://github.com/anycable/anycable-go/blob/master/etc/statsd_fake.rb) locally.

Let's update the example to match the actual format.